arm-datalake-analytics/job/2016-11-01/swagger/job.json

```diff
@@ -0,0 +1,1070 @@
+        },
+        "totalRunningTime": {
+          "type": "string",
+          "format": "duration",
+          "readOnly": true,
+          "description": "the total time this job spent executing. This value should not be set by the user and will be ignored if it is."
+        },
+        "rootProcessNodeId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the ID used to identify the job manager coordinating job execution. This value should not be set by the user and will be ignored if it is."
+        },
+        "yarnApplicationId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the ID used to identify the yarn application executing the job. This value should not be set by the user and will be ignored if it is."
+        },
+        "yarnApplicationTimeStamp": {
+          "type": "integer",
+          "readOnly": true,
+          "format": "int64",
+          "description": "the timestamp (in ticks) for the yarn application executing the job. This value should not be set by the user and will be ignored if it is."
+        },
+        "compileMode": {
+          "type": "string",
+          "description": "the compile mode for the job.",
+          "enum": [
+            "Semantic",
+            "Full",
+            "SingleBox"
+          ],
+          "x-ms-enum": {
+            "name": "CompileMode",
+            "modelAsString": false
+          }
+        }
+      },
+      "description": "U-SQL job properties used when submitting and retrieving U-SQL jobs."
+    },
+    "HiveJobProperties": {
+      "x-ms-discriminator-value": "Hive",
+      "allOf": [
+        {
+          "$ref": "#/definitions/JobProperties"
+        }
+      ],
+      "properties": {
+        "logsLocation": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the Hive logs location"
+        },
+        "outputLocation": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the location of Hive job output files (both execution output and results)"
+        },
+        "statementCount": {
+          "type": "integer",
+          "format": "int32",
+          "readOnly": true,
+          "description": "the number of statements that will be run based on the script"
+        },
+        "executedStatementCount": {
+          "type": "integer",
+          "format": "int32",
+          "readOnly": true,
+          "description": "the number of statements that have been run based on the script"
+        }
+      },
+      "description": "Hive job properties used when submitting and retrieving Hive jobs."
+    },
+    "JobProperties": {
+      "discriminator": "type",
+      "required": [
+        "script",
+        "type"
+      ],
+      "properties": {
+        "runtimeVersion": {
+          "type": "string",
```

Comment:
Unfortunately not, it is a custom label string that is used for internal flighting, debugging and for private customer tests, the labels can grow forever, so maintaining this as an enum would become cost prohibitive.